bears/cmake/CMakeLintBear.py

```diff
@@ -21,11 +21,15 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         cmake_filter: str=None,
                          cmakelint_config: path=''):
         """
         :param cmakelint_config: The location of the cmakelintrc config file.
+        :param cmake_filter: The filter to be applied on cmake file.
```

Comment:
put the parameter documentation in the same order as the parameters in the declaration